core/types/transaction_signing.go

```diff
@@ -65,6 +65,9 @@ func MakeSigner(config *params.ChainConfig, blockNumber *big.Int) Signer {
 // have the current block number available, use MakeSigner instead.
 func LatestSigner(config *params.ChainConfig) Signer {
 	if config.ChainID != nil {
+		if config.CurieBlock != nil {
```

Comment:
Does this change anything? And if we add `Curie`, shouldn't we add `Bernoulli` and `Shanghai` as well?

Reply:
it doesn't change anything, but in `MakeSigner` we have `IsCurie` check. so I think adding this will more the two function logic more consistent